cluster/src/main/java/com/hartwig/pipeline/tertiary/amber/AmberCommandBuilder.java

```diff
@@ -11,7 +11,7 @@
     private static final String MAIN_CLASS = "com.hartwig.hmftools.amber.AmberApplication";
 
     private AmberCommandBuilder(final ResourceFiles resourceFiles) {
-        super("amber", Versions.AMBER, MAX_HEAP, MAIN_CLASS, JAR);
+        super("amber", Versions.AMBER, MAX_HEAP, JAR);
```

Comment:
Can we remove `MAIN_CLASS`? Also I question know what the other constants add when they're used only once and their contents are pretty self-explanatory.